actors/evm/src/interpreter/system.rs

```diff
@@ -311,6 +350,16 @@ impl<'r, RT: Runtime> System<'r, RT> {
             .context_code(ExitCode::USR_SERIALIZATION, "failed to decode state")?
             .context_code(ExitCode::USR_ILLEGAL_STATE, "state not in blockstore")?;
 
+        let transient_data_lifespan = state.transient_data.unwrap().transient_data_lifespan;
+        let current_lifespan = Self::get_current_transient_data_lifespan(self.rt);
+        if current_lifespan == transient_data_lifespan {
+            self.transient_slots
+                .set_root(&state.transient_data.unwrap().transient_data_state.unwrap())
+                .context_code(ExitCode::USR_ILLEGAL_STATE, "transient_state not in blockstore")?;
+        } else {
+            self.transient_slots.clear();
```

Comment:
Note: if we get here, the transient slots _should_ already be empty. Although I guess it can't hurt to call this explicitly.

Reply:
removing the options from transient_data_state and transient_data_lifespan cleaned this up 978b49d1e0f1e50fbecf020cadcfb1b5b8c4275a

Reply:
I think it is good to add - it ensures there's a sanitation / matching of the data from the store into System